packages/commonwealth/client/scripts/views/pages/members.tsx

```diff
@@ -1,232 +1,61 @@
-      const scrollHeight = $(document).height();
-
-      const scrollPos = $(window).height() + $(window).scrollTop();
-
-      if (scrollPos > scrollHeight - 400 && !this.profilesFinishedLoading) {
-        const lastLoadedProfileIndex = this.numProfilesLoaded;
-
-        const newBatchSize = Math.min(
-          DEFAULT_MEMBER_REQ_SIZE,
-          this.membersLoaded.length - lastLoadedProfileIndex
-        );
-
-        const newBatchEnd = lastLoadedProfileIndex + newBatchSize;
-
-        for (let i = lastLoadedProfileIndex; i < newBatchEnd; i++) {
-          const member = this.membersLoaded[i];
-          const profileInfo: ProfileInfo = {
-            profile: app.profiles.getProfile(member.chain, member.address),
-            postCount: member.count,
-          };
-          this.profilesLoaded.push(profileInfo);
-        }
-
-        this.numProfilesLoaded += newBatchSize;
-        this.redraw();
-      }
-    }, 400);
-
-    const {
-      membersLoaded,
-      numProfilesLoaded,
-      profilesFinishedLoading,
-      profilesLoaded,
-      totalMembersCount,
-    } = this;
-
-    return (
-      <Sublayout>
-        <div className="MembersPage">
-          <CWText type="h3" fontWeight="medium">
-            {totalMembersCount ? `Members (${totalMembersCount})` : 'Members'}
-          </CWText>
-          <div className="header-row">
-            <CWText type="h5">Member</CWText>
-            <CWText type="h5">Posts / Month</CWText>
-          </div>
-          <Virtuoso
-            className="members-container"
-            style={{ height: 300 }}
-            data={profilesLoaded}
-            itemContent={(index, profileInfo) => {
-              return (
-                <div className="member-row" key={index}>
-                  <User
-                    user={profileInfo.profile}
-                    showRole
-                    hideAvatar={false}
-                    linkify={true}
-                  />
-                  <CWText>{profileInfo.postCount}</CWText>
-                </div>
-              );
-            }}
-            endReached={this.onScroll}
-            overscan={200}
-          />
-
-          <div className="infinite-scroll-wrapper">
-            {profilesFinishedLoading ? (
-              <CWText className="infinite-scroll-reached-end-text">
-                Showing all {membersLoaded.length} community members
-              </CWText>
-            ) : (
-              <CWSpinner size="large" />
-            )}
-          </div>
-        </div>
-      </Sublayout>
-    );
-  }
-}
+const MembersPage = () => {
+  const [membersList, setMembersList] = React.useState<Array<MemberInfo>>([]);
+
+  const activeInfo = app.chain.meta;
+
+  React.useEffect(() => {
+    const fetch = async () => {
+      await activeInfo.getMembers(activeInfo.id);
+
+      const profiles: Array<MemberInfo> = activeInfo.members.map((role) => ({
+        profile: app.profiles.getProfile(role.address_chain, role.address),
+      }));
```

Comment:
Can we sort this by roles? I don't like how "moderator" is all the way at the bottom. It should be:
Admins
Mods
everyone else